src/CFixer/main.cpp

```diff
@@ -6,14 +6,43 @@
 //#
 
 #include <quasarapp.h>
+#include <iostream>
+
+void helpCall();
 
 int main(int argc, char *argv[]) {
 
     if (!QuasarAppUtils::Params::parseParams(argc, argv)) {
-        QuasarAppUtils::Params::log("wrong parametrs", QuasarAppUtils::Warning);
-        exit(0);
+        helpCall();
+        return 1;
+    }
 
+    if (QuasarAppUtils::Params::isEndable("h") || QuasarAppUtils::Params::isEndable("help")) {
+        helpCall();
+        return 0;
     }
 
     return 0;
 }
+
+
+void helpCall(){
+
+    QuasarAppUtils::Help::Charters help = {
+        {
+            "Part 0 General", {
+                {"h or help", "These arguments represent a helper call that describes the functionality of each method"},
+                {"-sourceDir", "This arrgument sets path to the source directory. By default it is sourceDir = PWD"},
+                {"-sign", "This argument sets path to the sign patern. This is a required argument"},
+                {"-currentOwner", "This argument sets name of the current owner of the code."},
```

Comment:
i think you need to add description of options arguments
```suggestion
                {"-sourceDir path/to/source/dir", "This argument sets path to the source directory. By default it is sourceDir = PWD"},
                {"-sign path/to/sign/file", "This argument sets path to the sign patern. This is a required argument"},
                {"-currentOwner ownerName", "This argument sets name of the current owner of the code."},
```